tools/html-designer.rs

```diff
@@ -0,0 +1,150 @@
+// Copyright (C) 2022 Red Hat
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
this repo is under a gpl-3 license, any objection to changing this?

Reply:
sure.